### PR TITLE
Dedicate CLI to module for raco command

### PR DESCRIPTION
**Pull request by zyrolasting**
Run `raco file-watchers -h` for usage details.

### CODE REVIEW

File: info.rkt [hunk 1]
```diff
@@ -7,3 +7,5 @@
 (define pkg-desc "Recursive file system watching threads")
 (define version "0.1")
 (define pkg-authors '(Sage Gerard))
+(define raco-commands
+  '(("file-watchers" file-watchers/cli "Monitor files using file-watchers" 100)))
```
Comment (bennn) [inline info.rkt]: 100 seems high, I'd go with `#f`

File: cli.rkt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     [else "robust"]))
+
+(define method (case normalized-method-string
+   [("robust")    robust-watch]
+   [("intensive") intensive-watch]
+   [("apathetic") apathetic-watch]
+   [else          robust-watch]))
+
+(when (not (equal? normalized-method-string (method-string)))
+  (printf "Unrecognized method: ~a. Falling back to robust watch.~n" method-string))
```
Comment (bennn) [inline cli.rkt]: `(method-string)`

File: main.rkt [hunk 3]
```diff
@@ -63,7 +19,7 @@
                               (-> list? any)
                               (-> list? any)
                               (-> path? thread?))
-                             void?)]))
+                             thread?)]))
```
Comment (bennn) [inline main.rkt]: Oh, the doc contract for `watch-directories` needs an update

**Comment by zyrolasting** [comment]
Thanks @bennn!

**Pull request merged**